### ISSUE TITLE
Paths to fonts getting messed up in 1.1.17

### ISSUE THREAD

**Issue by punchcutter** [justvanrossum/fontgoggles#120]
Something seems odd with the relative paths in 1.1.17. I just opened 1.1.16, dragged in a font from Downloads and saved into `~/Desktop/fontgoggles/test.gggls`. 
The path to the font file is then saved as 
`../../Downloads/somefont.otf`
Doing the same thing in 1.1.17 makes the path 
`../Downloads/somefont.otf` which resolves to `~/Desktop/Downloads/somefont.otf` which makes no sense. Is there a reason the paths aren't saved as absolute paths?

**Comment by justvanrossum** [comment]
Hm, I am not aware of any change that could have changed this behavior, I will have to look into that.

The reason to store relative paths is for the (in my view) common use case that you store the .gggls file somewhere near the fonts, as a project file, not unlike .designspace, so one can move the entire project. The downside is that moving the .gggls file invalidates it's paths.

I don't know how to do this better unless I store macOS Alias data into the file, but that's probably what should be done.

**Comment by justvanrossum** [comment]
I just tried your steps and I can't reproduce it.

**Comment by punchcutter** [comment]
I know, I can't really reproduce it either. Except that it happened in a bunch of files for no apparent reason. And it will happen to some fonts, but not all in the same file. It's not like there's anything I could be doing to make it happen. I just open FontGoggles, drag in some fonts, save that as a project or maybe don't save, quit FontGoggles sometimes, reopen it. I haven't moved the .gggls files or the fonts at all. The only thing I did between everything being fine and weirdness was I restarted the machine at some point. Then installed 1.1.17 and suddenly had this weirdness. 

About relative paths...for me I have no need to store the .gggls file anywhere near the fonts because mostly I'm doing a lot of comparisons between different fonts in different locations. But I'm also not moving the .gggls files around so relative paths are also totally fine.
